src/ExpressionBuilder.php

```diff
@@ -77,6 +77,12 @@ public function isNull(string $field)
         return new Comparison($field, Comparison::EQ, new Value(null));
     }
 
+    /** @return Comparison */
+    public function isNotNull(string $field)
```

Comment:
```suggestion
    public function isNotNull(string $field): Comparison
```

Reply:
I was not sure whether it would be preferable to include the return type in the DockBloc for consistency with other methods or to add it as a native type as the method is a new one.